packages/@dcl/inspector/src/components/Renderer/Shortcuts/Shortcuts.tsx

```diff
@@ -16,6 +16,8 @@ import { Props } from './types'
 import './Shortcuts.css'
 
 const ICON_SIZE = 18
+const isMac = /Mac|iPhone|iPod|iPad/.test(navigator.userAgent)
```

Comment:
wdyt about moving this validation to a utils/helper file?

Reply:
I don' think it's necessary for now. It's a one-liner check. We should do it if we keep using it in other places but for now I think it's ok